paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -467,6 +467,7 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
                     address = address.value,
                 )
             ),
+            paymentMethodOptionsParams = null,
```

Comment:
It feels to me like we should set the `paymentMethodOptionsParams` here instead of in the `ConfirmStripeIntentParamsFactory`. Same goes for `card` and `link`.

It’s a little odd right now that we do it in different places for card/us_bank/link and blik.

Reply:
the issue with this is we don't have access to CustomerRequestedSave which is used to set the SetupFutureUsage

```
val setupFutureUsage = when (paymentSelection.customerRequestedSave) {
                CustomerRequestedSave.RequestReuse -> SetupFutureUsage.OffSession
                CustomerRequestedSave.RequestNoReuse -> SetupFutureUsage.Blank
                CustomerRequestedSave.NoRequest -> null
            }
```

Let me know if you see a way around this!